### PR TITLE
Compound interest

### LINKED ISSUE #1359: Write a F# program to calculate compound interest
### Description

Write a F# program to calculate compound interest

Compound interest = principal amount * (1 + rate of interest) ^ time

```
Principal : 1000
Rate      : 10
Time      : 5

Compound interest : 1610.51
```
<details>
<summary>How to contribute</summary>

- Save the solution in `CalculateCompoundInterest.fs` file
- Add `CalculateCompoundInterest.fs` file in `calculate-compound-interest` folder

</details>

### PR DESCRIPTION
**Pull request by anandfresh**
### Why:

Closes codinasion/codinasion#1064

### CODE REVIEW

File: program/calculate-compound-interest/CalculateCompoundInterest.fs [hunk 1]
```diff
@@ -0,0 +1,6 @@
+let compoundInterest principal interest time : float =
+  principal * (1.0 + interest) / time
```
Comment (victoriacheng15) [inline program/calculate-compound-interest/CalculateCompoundInterest.fs]: need to check the formula here

**Review by victoriacheng15 — CHANGES_REQUESTED** [review]

File: program/calculate-compound-interest/CalculateCompoundInterest.fs (hunk 1 above)
Comment (anandfresh) [reply program/calculate-compound-interest/CalculateCompoundInterest.fs]: @victoriacheng15 can you retest

Comment (victoriacheng15) [reply program/calculate-compound-interest/CalculateCompoundInterest.fs]: It should be `1610.51` but I got `161051000`
image

Comment (anandfresh) [reply program/calculate-compound-interest/CalculateCompoundInterest.fs]: @victoriacheng15 can you retest now

**Review by victoriacheng15 — APPROVED** [review]
Thank you for contributing!

**Pull request merged**